reposerver/repository/repository.go

```diff
@@ -222,12 +224,19 @@ func getHelmRepos(repositories []*v1alpha1.Repository) []helm.HelmRepository {
 }
 
 func helmTemplate(appPath string, env *v1alpha1.Env, q *apiclient.ManifestRequest) ([]*unstructured.Unstructured, error) {
+	var baseDirectoryPath string
+	if q.HelmOptions != nil && q.HelmOptions.DirectoryEnforcerLevel == v1alpha1.EnforcerLevelStrict {
```

Comment:
Hm, should we explicitly check for valid values as defined in `pkg/apis/application/v1alpha1/types.go`, except for just ignoring everything else besides `strict`?

Reply:
Good point, I'll change this.